docs/compatibility.md

```diff
@@ -1,263 +1,187 @@
+|   - extra_hosts              |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - labels                   |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - network                  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - shm_size                 |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - target                   |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - cap_add                    |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  C  |  C  |  C  |  C  |  C  |  C  |
+| - cap_drop                   |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  C  |  C  |  C  |  C  |  C  |  C  |
+| - cpu_shares                 |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - cpu_quota                  |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - cpuset                     |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - command                    |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - configs                    |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - cgroup_parent              |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - container_name             |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - credential_spec            |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+|   - file                     |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+|   - registry                 |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - deploy                     |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|   - endpoint_mode            |  S  |  S  |  S  |  S  |  S  |     |     |     |     |     |     |     |     |
+|     - dnsrr                  |  S  |  S  |  S  |  S  |  S  |     |     |     |     |     |     |     |     |
+|     - vip                    |  S  |  S  |  S  |  S  |  S  |     |     |     |     |     |     |     |     |
+|   - labels                   |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|   - mode                     |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|   - placement                |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|   - replicas                 |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
+|   - resources                |  SK |  SK |  SK |  SK |  SK |  SK |  SK |  SK |     |     |     |     |     |
```

Comment:
Supported in Compose if *compatibility mode* is enabled: https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-mode

Reply:
Fixed.